scripts/tokenizer.py

```diff
@@ -4,39 +4,92 @@
 from pygments.lexers import Python3Lexer
 from pygments.token import Token
 from random import randint
+import re
 
 
 class Tokenizer:
     TOKENS = list()
+    VARS = list()
+    QUOTES = ["'", '"', '"""', "'''"]
+    BOOLS = ['True', 'False']
```

Comment:
I think BOOLEAN would be better in this case, but it's not necessary.

Reply:
Right. Made changes.